bin/lib/zopen-build

```diff
@@ -989,6 +997,12 @@ resolveCommands()
   if [ "${ZOPEN_INSTALL_DIR}x" = "x" ]; then
     export ZOPEN_INSTALL_DIR="${HOME}/zopen/prod/${ZOPEN_NAME}"
   fi
+  if [ "${ZOPEN_LOG_DIR}x" = "x" ]; then
+    export ZOPEN_LOG_DIR="${ZOPEN_ROOT}/log"
+  fi
+
+  [[ -d ${ZOPEN_LOG_DIR} ]] || mkdir -p ${ZOPEN_LOG_DIR}
+
   if [ "${ZOPEN_CONFIGURE_OPTS}x" = "x" ]; then
     export ZOPEN_CONFIGURE_OPTS="--prefix=${ZOPEN_INSTALL_DIR} ${ZOPEN_EXTRA_CONFIGURE_OPTS}"
```

Comment:
I missed this with adding cmake support.  I'll fix this in another PR.  cmake uses `--install-prefix <dir>`.